crates/release-automation/src/lib/tests/workspace_mocker.rs

```diff
@@ -270,7 +271,59 @@ impl WorkspaceMocker {
 }
 
 /// Expected changelog after aggregation.
-pub fn example_workspace_1_aggregated_changelog() -> String {
+pub fn example_workspace_1_aggregated_changelog(crate_order: Vec<String>) -> String {
```

Comment:
certainly more fancy than i anticipated this function to ever get :smile:

Reply:
It's actually more similar to the way the code it's testing works than it was by taking the ordered workspace. There are other tests for checking order and this one can just check that the right content is making it through